ckb-importer/src/main/java/com/hartwig/serve/ckb/datamodel/treatmentapproaches/RelevantTreatmentApproachesFactory.java

```diff
@@ -26,8 +27,9 @@ public static List<DrugClassTreatmentApproach> extractDrugTreatmentApproaches(@N
         for (TreatmentApproachInfo treatmentApproachInfo : treatmentApproachInfos) {
             DrugClassTreatmentApproach resolvedRelevantTreatmentApproaches =
                     resolveDrugTreatmentApproaches(ckbJsonDatabase, treatmentApproachInfo);
-            relevantTreatmentApproach.add(resolvedRelevantTreatmentApproaches);
-        }
+            if (resolvedRelevantTreatmentApproaches != null) {
+                relevantTreatmentApproach.add(resolvedRelevantTreatmentApproaches);
+            }        }
```

Comment:
Would be good to fix the funky formatting :)

Reply:
solved